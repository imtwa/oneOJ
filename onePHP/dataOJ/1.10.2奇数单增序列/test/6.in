65
377 175 485 191 352 301 442 334 229 122 412 213 396 443 119 473 276 191 229 319 319 290 232 197 296 105 388 377 133 211 379 253 204 177 301 280 261 337 463 134 127 390 310 458 202 188 90 432 208 299 200 127 394 415 462 260 461 165 163 178 209 412 237 158 171 