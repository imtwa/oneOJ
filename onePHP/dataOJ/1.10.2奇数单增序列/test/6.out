105,119,127,127,133,163,165,171,175,177,191,191,197,209,211,213,229,229,237,253,261,299,301,301,319,319,337,377,377,379,415,443,461,463,473,485
